torch/nativert/common/Conv.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <cstdint>
+#include <optional>
+#include <string_view>
+
+#include <c10/macros/Export.h>
+
+namespace torch::nativert {
```

Comment:
Why does it need to be in nativert namespace? `at::utils::` feels like a more appropriate namespace. And there probably already a function there